catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/QueryFilterTransformerProvider.java

```diff
@@ -53,10 +55,9 @@ public synchronized void bind(ServiceReference<QueryFilterTransformer> reference
     for (QName namespace : namespaces) {
       queryFilterTransformerMap.put(namespace, transformer);
       List<String> typeNames = getTypeNames(reference);
-      if (CollectionUtils.isNotEmpty(typeNames)) {
-        for (String typeName : typeNames) {
-          typeNameQNameMap.put(typeName, namespace);
-        }
+
```

Comment:
✏️  The same can be done on line 74 below

Reply:
done